extra/1-currency-conversion.js

```diff
@@ -15,7 +17,10 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(price) {
+	const perc99 = (price / 100) * 99;
```

Comment:
This works perfectly 👍, my only comment would be your naming of your variable, `perc99` is somewhat clear but it would be even better if you use more descriptive variable names such as `percent99` or maybe even something like `const deductedPrice = ...`

If you imagine a code block that is 100-200 lines long, the more descriptive your variables are, the easier it is for the reader to understand what is happening. `perc99` is somewhat descriptive but it's even better to be more descriptive than that :)

Reply:
ok I'll do that. thanks